### ISSUE TITLE
Page Break in Drasil GlassBR SRS

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#822]
This is a small issue, but there is a page break between the table of contents and the "Reference Material" section in Drasil (left) that is missing from caseStudies.

![page break](https://user-images.githubusercontent.com/35857611/42575575-e16ffd8a-84ee-11e8-90bc-894594857288.png)

**Comment by szymczdm** [comment]
I think separating the ToC with a page break looks cleaner.

**Comment by smiths** [comment]
Yes, the page break looks better.  @samm82, please add it to the case study.
